src/components/DonatorForm/index.jsx

```diff
@@ -0,0 +1,123 @@
+import React, { useState } from "react";
+import { Container, Form, Button, Col, Row } from "react-bootstrap";
+import db from "../../firebaseConfig";
+import "./index.css";
+
+export default function DonatorForm() {
+  const [donatorPostData, setDonatorPostData] = useState({});
+
+  const handlePostChange = (e, key) => {
+    console.log(e.target.value);
+    setDonatorPostData({ ...donatorPostData, [key]: e.target.value });
+    console.log(donatorPostData);
+  };
+
+  const addDonatorPost = (e) => {
+    e.preventDefault();
+    console.log("yeeyyyyyy");
+    db.collection("donatorPost").add({
+      title: donatorPostData.title,
+      type: donatorPostData.type,
+      description: donatorPostData.description,
+      target: donatorPostData.target,
+      profileName: donatorPostData.profileName,
+      about: donatorPostData.about,
+      address: donatorPostData.address,
+      date: donatorPostData.date,
+    });
+  };
+
+  return (
+    <div>
+      <Container>
+        <Form onSubmit={addDonatorPost}>
+          <Form.Group>
+            <Form.Label>Donation Type</Form.Label>
+            <Form.Control
+              as="select"
+              onChange={(e) => handlePostChange(e, "type")}
+            >
+              <option value="technology">Technology</option>
```

Comment:
Can you use i18next for all the text to reduce the amount of work we have in the final couple of days

Reply:
PTAL